cmd/influx/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/urfave/cli/v2"
```

Comment:
Risking an innovation token on `urfave/cli` over `spf13/cobra`. `cobra` has always felt more heavyweight/verbose to me than a CLI parser needs to be. `cli` looks very similar to the `kit/cli` convenience wrapper in `influxdb`.

If we hit a wall with it, we can fall back to using `cobra` and copy the code from `influxdb`.